### ISSUE TITLE
LOW_LEVEL_MODULE warnings should be limited to SDK-based extensions

### ISSUE THREAD

**Issue by palant** [mozilla/addons-linter#662]
I'm trying this linter on the WebExt build of [Easy Passwords](https://github.com/palant/easypasswords/) (produced via `gulp webext`). The following warning shows up:

```
LOW_LEVEL_MODULE   Usage of low-level or   Your add-on uses an interface which bypasses the   background.js   567    12    
                   non-SDK interface       high-level protections of the add-on SDK. This                                  
                                           interface should be avoided, and its use may                                    
                                           significantly complicate your review process                               
```

While `background.js` indeed calls `require("chrome")`, this build isn't based on the Add-on SDK - it cannot possibly bypass high-level protections. In fact, `background.js` has been generated via browserify and has a fake implementation of the `chrome` module (providing `Cu.importGlobalProperties()` which doesn't need to do anything).

**Comment by tofumatt** [comment]
Thanks for the report! Looks like a false positive--any chance you could
send a gist/attach a copy of the file that's causing this warning so I can
test against it and fix it?

- tofumatt

**Comment by palant** [comment]
Attached: [easypasswords-1.0.6.xpi](https://github.com/mozilla/addons-linter/files/261015/easypasswords-1.0.6.zip)

**Comment by muffinresearch** [comment]
This is planned to be completely removed, marking as a duplicate of https://github.com/mozilla/addons-linter/issues/1063
